mbuild/formats/hoomd_snapshot.py

```diff
@@ -134,10 +133,38 @@ def to_hoomdsnapshot(
     ReferenceValues : namedtuple
         Values used in scaling
 
-
-    Notes
+    Note
```

Comment:
I think this wants to be in one big Notes heading. That's just the standard way to notate things so we want to stick with that. I think something like this would be good:
```python
        Notes
        -----
        This method does not create hoomd forcefield objects...

        About Units: This method operates on a Parmed.Structure...

```